configuration/messageproperties/messages_en.properties

```diff
@@ -1999,7 +2000,6 @@ mirebalais.dispensing.medicationDuration=Duration
 mirebalais.dispensing.medicationAmount=Amount
 mirebalais.dispensing.medicationInstructions=Administration Instructions
 mirebalais.dispensing.prescriber=Prescriber (optional)
-pihcore.medicationDispensingHistory=Medication Dispensing History
```

Comment:
Don't delete this line @louidorjp .  I think it was added recently by @mogoodrich